include/Simulator.hpp

```diff
@@ -80,7 +80,7 @@ class Simulator {
 
     [[nodiscard]] virtual std::size_t getMatrixActiveNodeCount() const { return dd->mUniqueTable.getActiveNodeCount(); }
 
-    [[nodiscard]] virtual std::size_t countNodesFromRoot() const { return dd->size(rootEdge); }
+    [[nodiscard]] virtual std::size_t countNodesFromRoot() { return dd->size(rootEdge); }
```

Comment:
Is the `size()` function in the DD package really not `const`?

Reply:
The `size` function is `const` but the `countNodesFromRoot` in `DeterministicNoiseSimulator` is not.

Reply:
Ah, makes ~sense.